src/Oscoin/Data/Ledger.hs

```diff
@@ -102,6 +113,9 @@ data Account c = Account
     -- made from this account.
     }
 
+instance (Eq (PublicKey c)) => Eq (Account c) where
+    (==) a b = accountAddr a == accountAddr b
```

Comment:
Indeed, I reverted to deriving all instances when possible. The only one I can't derive is `Project` because of the functions in there: 2eb1d515b0a186f9bea7ef657ed44680479c7adb

Reply:
As the data model gains clarity, I might separate the "Project" that is stored in the state, and the "Project" that is passed to the contract handler, at which point I'll revisit this.